src/main/i18n/gradle-enterprise-bamboo-plugin.properties

```diff
@@ -18,3 +18,4 @@ gradle-enterprise.config.server=Gradle Enterprise server URL
 gradle-enterprise.config.shared-credential-name.description=The name of the shared credential of type &lsquo;Username and password&rsquo; containing the access key for authenticating with the Gradle Enterprise server.
 gradle-enterprise.config.shared-credential-name=Shared credential name
 gradle-enterprise.config.access-key-info=The access key must be in the <b>&lt;server host name&gt;=&lt;access key&gt;</b> format. For more details please refer to the <a href="https://docs.gradle.com/enterprise/gradle-plugin/#manual_access_key_configuration" target="_blank">documentation</a>.
+gradle-enterprise.config.auto-injection-info=<div>To enable instrumentation for Gradle builds set the Gradle Enterprise Server URL and the desired version of the Gradle Enterprise Gradle plugin.</div><div>To enable instrumentation for Maven builds set the Gradle Enterprise Server URL and select <em>Enables Gradle Enterprise Maven extension auto-injection</em>.</div><div>For more details please refer to the <a href="https://github.com/gradle/gradle-enterprise-bamboo-plugin#configuration" target="_blank">documentation</a>.</div>
```

Comment:
For both the Maven and the Gradle lines, do not capitalize the `S` in `Server`:

...Gradle Enterprise server URL...

This will then match the label of that field.

Reply:
Done.